client/devdocs/design/index.jsx

```diff
@@ -153,9 +154,14 @@ class DesignAssets extends React.Component {
 				<DocumentHead title="UI Components" />
 
 				{ component ? (
-					<HeaderCake onClick={ this.backToComponents } backText="All Components">
-						{ slugToCamelCase( component ) }
-					</HeaderCake>
+					<React.Fragment>
+						<HeaderCake onClick={ this.backToComponents } backText="All Components">
+							{ slugToCamelCase( component ) }
+						</HeaderCake>
+						{ config.isEnabled( 'devdocs/color-scheme-picker' ) && (
+							<ColorSchemePicker readmeFilePath="color-scheme-picker" />
```

Comment:
Should we display this color scheme picker even if the component shown is the color picker itself?